docs/upgrade/README.md

```diff
@@ -23,7 +23,52 @@ Details of how to upgrade can be found for each deployment model:
 - [Docker](../install/docker/README.md#upgrade)
 - [Kubernetes](../install/kubernetes/README.md#upgrade)
 
-## Specific requirements for a given target version
+### Upgrading to 1.15
```

Comment:
```suggestion
### Upgrading to 2.0.0
```

Reply:
This also needs a comment about this being specific to kubernetes

Reply:
Installation with Helm Chart is Kubernetes specific? Does it need more clarification?

Reply:
Actually looking at it again, I think this should all be moved over the end of docs/install/kubernetes/README.md#upgrade section since it is specific to the kubernetes upgrade

Reply:
Moved.